iocore/net/SSLClientUtils.cc

```diff
@@ -66,6 +66,7 @@ verify_callback(int preverify_ok, X509_STORE_CTX *ctx)
   netvc = SSLNetVCAccess(ssl);
 
   if (netvc != nullptr) {
+    netvc->callHooks(TS_EVENT_SSL_SERVER_VERIFY_HOOK);
```

Comment:
The use case we have which triggered the addition of this verification hook is that the plugin needs to verify some extra information in addition to what ATS and Openssl is verifying. I think we need a separate config for prioritizing plugin validation over OpenSSL or ATS. What are your thoughts?